pipelines/pipelines/cre-import.yaml

```diff
@@ -3,16 +3,16 @@ kind: Pipeline
 metadata:
   name: cre-import
   labels:
-    app.kubernetes.io/created-by: byon
+    app.kubernetes.io/created-by: CustomRuntimeEnvironment
```

Comment:
Should we specify 'CRE **controller**' ? Not sure if that cause confusion or not.

Reply:
Honestly I was just doing a "literal update" here (byon -> CNBi -> CRE)

Looking closer now, according to Kubernetes' [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) it seems this should be `part-of` instead of `created-by` I think. And for the value, it should probably be `meteor-operator`.

Reply:
actually, if it is a piece of software: https://en.wikipedia.org/wiki/Common_Platform_Enumeration ? ;)

but I agree with the part-of argument